src/lib/compute_text_alternative_test.ts

```diff
@@ -157,4 +157,25 @@ describe('The computeTextAlternative function', () => {
     const elem = document.getElementById('test')!;
     expect(computeTextAlternative(elem).name).toBe('Flash the screen 2 times.');
   });
+
+  // http://wpt.live/accname/name_checkbox-label-embedded-menu-manual.html
```

Comment:
How is this link relevant?

Reply:
The unit test on the next lines (162 - 180) is found at this link.